howso/engine/trainee.py

```diff
@@ -95,6 +96,8 @@ class Trainee(BaseTrainee):
         The Howso client instance to use.
     overwrite_existing : bool, default False
         Overwrite existing trainee with the same name (if exists).
+    runtime : TraineeRuntimeOptions, optional
```

Comment:
Lets add the deprecation notes in this docstring too, since this is the main interface users would reference.

Reply:
done